builders/build/eth-api/dev-env/brownie.md

```diff
@@ -37,13 +37,15 @@ You will need to install Brownie and create a Brownie project if you don't alrea
     python3 -m pipx ensurepath
     ```
 
-3. [Install Brownie using `pipx`](https://eth-brownie.readthedocs.io/en/stable/install.html){target=_blank}
+3. [Install Brownie using `pipx`](https://eth-brownie.readthedocs.io/en/stable/install.html){target=_blank}, which is used to run executables installed locally in your project. Brownie will be installed into a virtual environment and be available directly from the command line.
     ```bash
     pipx install eth-brownie
-    ```
+    ``` 
+If you're running Ubuntu, you should use the following command to avoid installation errors.
```

Comment:
```suggestion

    !!! note
        A common error while installing Brownie on Ubuntu is:
        
        ```text
        pip seemed to fail to build package:
            pyyaml==5.4.1

        Some possibly relevant errors from pip install:
            error: subprocess-exited-with-error
            AttributeError: cython_sources

        Error installing eth-brownie.
        ```

        This can be resolved by using the following command:
```